spring-cloud-stream-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -83,51 +87,79 @@ Spring Cloud Stream provides a number of abstractions and primitives that simpli
+Taking the sink example from the previous section, providing the `spring.cloud.stream.bindings.input.destination=raw-sensor-data` property to the application will cause it to read from the `raw-sensor-data` Kafka topic, or from a queue bound to the `raw-sensor-data` exchange in Rabbit MQ.
+
+Spring Cloud Stream will automatically detect and use a binder that is found on the classpath, so you can easily use different types of middleware with the same code, just by including a different binder at build time.
+For more complex use cases, Spring Cloud Stream also provides the ability of packaging multiple binders within the same application and choosing what type of binder should be used at runtime, and even if multiple binders should be used at runtime for different channels.
 
-This can mean, for example, dynamically choosing the destinations that these channels connect to at runtime (e.g. Kafka topics or Rabbit MQ exchanges), via external configuration properties in any form that is supported by Spring Boot (application arguments, environment variables, `application.yml` files, etc). Taking the source example from the previous section, providing the `spring.cloud.stream.bindings.output.destination=raw-sensor-data` property to the application will cause it to write to the `raw-sensor-data` Kafka topic, or the `orders` exchange in Rabbit.
+==== Fat JAR
 
-On the other hand, since the application is not middleware-specific, Spring Cloud Stream will automatically detect and use a binder that is found on the classpath, so you can easily use different types of middleware with the same code, just by including a different binder at build time. For more complex use cases, Spring Cloud Stream also provides the ability of packaging multiple binders within the same application and choosing what type of binder should be used at runtime, and even if multiple binders should be used at runtime for different channels.
+Spring Cloud Stream applications can be run in standalone mode from your IDE for testing. To run in production you can create an executable (or "fat") JAR using the standard Spring Boot tooling provided for Maven or Gradle.
 
 === Persistent publish subscribe and consumer groups
 
-Spring Cloud Stream is a library focusing on building message-driven microservices, and more specifically stream processing applications. In such scenarios, communication between different logical applications follows a publish-subscribe pattern, with data being broadcast through shared topics. This can be seen in the following picture, which shows a typical deployment for a set of interacting Spring Cloud Stream applications.
+For message-driven microservices and stream processing scenarios, communication between different applications typically follows a publish-subscribe pattern, with data being broadcast through shared topics.
```

Comment:
No need to qualify with "For message-driven microservices and stream processing scenarios,"